logrotate.h

```diff
@@ -45,9 +45,9 @@ struct logInfo {
     char *oldDir;
     enum { ROT_HOURLY, ROT_DAYS, ROT_WEEKLY, ROT_MONTHLY, ROT_YEARLY, ROT_SIZE
             } criterium;
-    unsigned long long threshhold;
-    unsigned long long maxsize;
-    unsigned long long minsize;
+    size_t threshhold;
```

Comment:
You are right, off_t would have been better type with printf using %jd && (intmax_t) cast. Done that way.